srcpkgs/igt-gpu-tools/template

```diff
@@ -0,0 +1,27 @@
+# Template file for 'igt-gpu-tools'
+pkgname=igt-gpu-tools
+version=1.23
+revision=1
+build_style=meson
+hostmakedepends="flex gtk-doc pkg-config swig"
+makedepends="cairo-devel libXrandr-devel libXv-devel libkmod-devel
+ libunwind-devel procps-ng-devel python3-devel libressl-devel"
+short_desc="Tools for development and testing of the Intel DRM driver"
+maintainer="Juan RP <xtraeme@voidlinux.eu>"
+license="MIT"
+homepage="https://cgit.freedesktop.org/xorg/app/intel-gpu-tools/"
```

Comment:
You could probably change the homepage over to https://gitlab.freedesktop.org/drm/igt-gpu-tools if you want.

Reply:
Good idea, didn't realize cgit is mirror now.